src/lib/config.rs

```diff
@@ -0,0 +1,24 @@
+use figment::{
+    providers::{Env, Format, Toml},
+    Error, Figment,
+};
+use serde::Deserialize;
+use serenity::model::id;
+
+#[derive(Deserialize, Clone)]
+pub struct Config {
+    pub discord_token: String,
+    pub spotify_username: String,
+    pub spotify_password: String,
+    pub discord_user_id: id::UserId,
+}
+
+impl Config {
+    pub fn new() -> Result<Self, Error> {
+        let config: Config = Figment::new()
+            .merge(Toml::file("config.toml"))
+            .merge(Env::prefixed("AOEDE_").map(|v| v.to_string().to_lowercase().into()))
```

Comment:
Is it possible to use figment without prefixing env vars to avoid breaking changes?

Reply:
[sure](https://docs.rs/figment/latest/figment/providers/struct.Env.html#method.raw)!